vnext/template/cs-lib/proj/Directory.Build.props

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Should this file be in the csproj filters?

Reply:
Nope (note the `Condition` :) ).

For instance, C# projects already have the `PackageReferences` capability.
It's only VCXPROJ files that need to add it explicitly.